Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GO_IMAGE=rancher/hardened-build-base:v1.23.10b1
+ARG GO_IMAGE=rancher/hardened-build-base:v1.23.11b1
```

Comment:
Should be safe to bump this to 1.24, we have been building runc with 1.24 in k3s for quite a while.